src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
```

Comment:
As mentioned in the comment for `Lottery`, if you decide to remove the no-argument constructor we can delete the getters and setters, as they are not used anymore